ambry-router/src/test/java/com/github/ambry/router/UndeleteManagerTest.java

```diff
@@ -373,14 +374,12 @@ private void assertDeleted(String blobId) throws Exception {
   }
 
   private void assertNotDeleted(String blobId) throws Exception {
-    router.getBlob(blobId, new GetBlobOptionsBuilder().build())
-        .get(AWAIT_TIMEOUT_MS, TimeUnit.MILLISECONDS);
+    router.getBlob(blobId, new GetBlobOptionsBuilder().build()).get(AWAIT_TIMEOUT_MS, TimeUnit.MILLISECONDS);
   }
 
   private void deleteBlobInAllServer(String blobId) throws Exception {
     // Delete this blob
-    router.deleteBlob(blobId, "serviceid")
-        .get(AWAIT_TIMEOUT_MS, TimeUnit.MILLISECONDS);
+    router.deleteBlob(blobId, "serviceid").get(AWAIT_TIMEOUT_MS, TimeUnit.MILLISECONDS);
```

Comment:
rm formatting edits. 
PR can be smaller if all formatting edits are moved to a different PR.